mk/build_optee_for_android.mk

```diff
@@ -0,0 +1,88 @@
+##########################################################
```

Comment:
Please rename this file to `aosp_optee.mk`. Also, wouldn't it be possible to use a regular `Android.mk` here instead that would be picked up by the Android build system?